macros/utils/command_type_utils.sql

```diff
@@ -0,0 +1,11 @@
+{% macro is_test_command(command) %}
```

Comment:
Maybe the macros shouldn't receive anything and read `flags.WHICH` themselves given that it's a global?

Reply:
done